### PR TITLE
Document new issues seen with Storage hosts

### PR DESCRIPTION
**Pull request by MoteHue**

### CODE REVIEW

File: doc/source/operations/rocky-linux-9.rst [hunk 1]
```diff
@@ -233,11 +233,14 @@ Potential issues
...
-  <https://bugs.launchpad.net/neutron/+bug/2042938>`__ for details. The issue
-  may be worked around after Neutron has been restarted by detaching then
-  reattaching the floating IP to the load balancer's virtual IP.
+-  When using Octavia load balancers, restarting Neutron causes load balancers
+   with floating IPs to stop processing traffic. See `LP#2042938
+   <https://bugs.launchpad.net/neutron/+bug/2042938>`__ for details. The issue
+   may be worked around after Neutron has been restarted by detaching then
+   reattaching the floating IP to the load balancer's virtual IP.
+
+-  If you are using hyper-convered Ceph, please also note the potential issues
```
Comment (mnasiadka) [inline doc/source/operations/rocky-linux-9.rst]: Converged

Comment (MoteHue) [reply doc/source/operations/rocky-linux-9.rst]: Done

**Comment by markgoddard** [comment]
How is it looking now @mnasiadka ?

**Comment by markgoddard** [comment]
Tox is moaning:

/home/runner/work/stackhpc-kayobe-config/stackhpc-kayobe-config/doc/source/operations/rocky-linux-9.rst:489: WARNING: Inline literal start-string without end-string.
/home/runner/work/stackhpc-kayobe-config/stackhpc-kayobe-config/doc/source/operations/rocky-linux-9.rst:489: WARNING: Inline literal start-string without end-string.

**Comment by MoteHue** [comment]
Tox is happy now :)

**Review by markgoddard — APPROVED** [review]

**Pull request merged**